agenthub/coact_agent/meta/executor_prompt.py

```diff
@@ -0,0 +1,37 @@
+EXECUTOR_PASS_CHECK = """Now, your role is to ensure the successful execution of actions in the global plan. Verify the results of these actions and compare them to the global plan. If they align, proceed to the next phase and output the action decision as ```move```. If discrepancies arise, you have two options:
+1) If you suspect issues with your local plan, output the action ```revise```, or
+2) If you suspect problems with the global planner's plan, trigger a request for replanning by outputting the action ```request```.
+If the actions align with the global plan, explain the reasons for this alignment. If discrepancies arise, provide detailed reasons for your action decision:
+- If you suspect issues with your local plan, explain why and use the action ```revise```.
+- If you suspect problems with the global planner's plan, explain why and use the action ```request```.
+Follow this structured output template:
+
+**Output:**
+- Output the action decision and reasons for your decision. Use the following format:
+    ```
+    Action: [action]
+    Reasons: [reasons]
+    ```
+Next, you will be provided with the expected state of this phase and the execution results of your local plan.
+"""  # FIXME: delete this as we may not use it
+
+EXECUTOR_FALSE_CHECK = """You have encountered an exception in the execution process. Your current responsibility is to meticulously inspect the execution results of actions and identify the root causes of these exceptions. You have two options:
+1) Suspect issues within your local plan and employ the action ```revise```, or
+2) Suspect problems with the global planner's plan and trigger a request for replanning by executing the action ```request```.
+Provide detailed reasons for your action decision:
+- If you suspect issues with your local plan, explain why you decide to use the action ```revise```.
+- If you suspect problems with the global planner's plan, explain why you decide to use the action ```request```.
+Follow this structured output template:
+
+**Output:**
+- Output the action decision and reasons for your decision. Use the following format:
+    ```
+    Action: [action]
+    Reasons: [reasons]
+    ```
+Next, you will be provided with the expected state of this phase and the execution exceptions during the execution of your local plan.
+"""
+
+EXECUTOR_SELF_REVISE = """Now, you have analyzed the situation and decided adjustments are needed to the local plan. Here is the reasons you proposed: {reasons}. Provide a revised plan using Page Operation Actions, and make sure to follow the format for action generation as mentioned earlier"""
```

Comment:
In the paper's code, the actions of the local agent are named "Page Operations Actions", but we don't name them here, so the LLM might not know what are these page operations? We only use "Page Operation Actions" here and below, as references to something previously defined. Perhaps it would be better to either define it or avoid to refer to this name?

Reply:
I don't use these meta prompts yet for the agents, just put it there as a reminder to use in case it doesn't follow the workflow properly. So didn't modify the original prompt from the repo. Will adjust when we actually use it